.github/workflows/release-cli.yml

```diff
@@ -0,0 +1,32 @@
+name: Release CLI
+
+on:
+  push:
+    tags:
+      - cli-v*
```

Comment:
The convention is to use `v*` tags (e.g. v1.0.0), however this is a mono-repo [publishing Docker images](https://github.com/guardian/devx-logs/blob/main/.github/workflows/release.yml), so add the `cli-` prefix to help distinguish what should be released.